apps/docs/content/components/Card/react.mdx

```diff
@@ -148,8 +148,8 @@ The color of the lit area corrosponds to the nearest active, primary accent colo
 
 ```jsx live
 <ThemeProvider colorMode="dark">
-  <Box padding="spacious" backgroundColor="default">
-    <Stack direction="horizontal" padding="none">
+  <Box backgroundColor="default" padding="condensed">
+    <Stack direction="vertical" padding="none">
```

Comment:
It looks a bit odd now being vertically stacked, and isn't representative of how we'd recommend using Card (we'd ask to go fullWidth to fill the blank space).

![Screenshot 2025-02-11 at 14 38 37](https://github.com/user-attachments/assets/2be23209-75c4-41a8-bf5c-880cad14c5a6)

Could you drop the first one here and just show one card with the style override?